macros/schema_tests/multi-column/expect_compound_columns_to_be_unique.sql

```diff
@@ -39,6 +39,7 @@ with validation_errors as (
         {% for column in columns -%}
         {{ column }}{% if not loop.last %},{% endif %}
         {%- endfor %}
+        ,count(*) as {{adapter.quote("n_records")}}
```

Comment:
slight nit: we do commas at the of lines in this repo, thanks!

Reply:
I assume you mean end.  On it.

Reply:
Do you need me to rebase the commits?

Reply:
All good, thanks!